python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -1233,3 +1263,47 @@ def _make_outputs(self, wcsf, visitSummaryTables, exposureInfo):
             catalogs[visit] = catalog
 
         return catalogs
+
+    def _output_model_params(self, wcsf):
```

Comment:
I'd call this `_compute_model_params` if it's returning them and not actually doing output itself.

Reply:
How about just `_get_model_params`?

Reply:
If it's just copying values and not really computing anything, ok.